create_mini_agasc_h5.py

```diff
@@ -5,12 +5,12 @@
 from astropy.table import Table
 
 
-example_file = '/data/agasc1p6/agasc/n0000/0001.fit'
+example_file = '/proj/sot/ska/data/agasc1p7/agasc/n0000/0001.fit'
```

Comment:
For this script you should use the version I created earlier that has a command line input to create either 1p6 or 1p7.

Reply:
I'm talking about the entire create_mini_agasc_h5.py script, not the `example_file`.  See: https://github.com/sot/agasc/blob/270f862458c10aaf5d1768e9711eb51b7f6fe083/create_mini_agasc_h5.py

From https://github.com/sot/agasc/pull/25/files.

Or am I still misunderstanding?

Reply:
Ah, OK, my mistake. I will use the version with a command line input.